verifact/forum/schema.py

```diff
@@ -118,7 +126,53 @@ def mutate_and_get_payload(
 
         return AnswerCreate(answer=answer)
 
+class VoteUpdate(ClientIDMutation):
+    vote = Field(VoteNode)
+
+    class Input:
+        credible = Boolean() # if null, remove vote!
+        answer_id = ID()
```

Comment:
I think this should be required as well. I don't think this mutation works without an `answer_id`